profiles/common/modules/features/newsletters/newsletters.install

```diff
@@ -32,3 +37,48 @@ function newsletters_disable() {
   multisite_drupal_toolbox_remove_block_context('site_wide', '0');
   drupal_set_message(t('Newsletters feature is now disabled on your site.'));
 }
+
+/**
+ * Update the title field of existing newsletters if any.
+ */
+function newsletters_update_7001(&$sandbox) {
+  // If this is the first pass through this update
+  // function then set some variables.
+  if (!isset($sandbox['total'])) {
+    $result = db_query("SELECT nid FROM {node} WHERE type='simplenews'");
+    $sandbox['total'] = $result->rowCount();
+    $sandbox['current'] = 0;
+
+    // Check if the content type has content translation.
+    $translation_support = variable_get('language_content_type_simplenews', 0);
+
+    if ($translation_support == 2) {
+      // Using content translation.
+      return;
+    }
+
+    // Activate entity translation.
+    variable_set('language_content_type_simplenews', ENTITY_TRANSLATION_ENABLED);
+
+    // Configure title_field and field_groups.
+    _newsletters_soft_config_title_field();
+  }
+
+  // How many nodes should be processed per pass.
+  $nodes_per_pass = 10;
```

Comment:
This is just the number of nodes that will be processed each time, just in case there is a site to update in which there are too many nodes to process.